### PR TITLE
Basic hello world with Django in Python

### PR DESCRIPTION
**Pull request by engelke**

### CODE REVIEW

File: python/cloud-run-django-hello-world/.idea/modules.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/cloud-run-django-hello-world.iml" filepath="$PROJECT_DIR$/.idea/django-hello-world.iml" />
```
Comment (simonz130) [inline python/cloud-run-django-hello-world/.idea/modules.xml]: Just verifying - is the dollar sign at the end of the variable "by design"?

File: python/cloud-run-django-hello-world/.vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+// launch.json contains information needed for the debugger to attach to  processes
+// See https://code.visualstudio.com/Docs/editor/debugging#_launch-configurations
+// for documentation about how to create and manage configurations
+{
+  "version": "0.2.0",
```
Comment (simonz130) [inline python/cloud-run-django-hello-world/.vscode/launch.json]: Again, flagging this file as we'll need changes here but I don't have the exact changes yet.

File: python/cloud-run-django-hello-world/cloudbuild.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  - --region
+  - ${_REGION} # e.g. us-central1
+  - --platform
+  - managed
+  - --allow-unauthenticated
+
+images:
+- gcr.io/${_PROJECT_ID}/${_SERVICE}-${COMMIT_SHA}
+
+substitutions:
```
Comment (simonz130) [inline python/cloud-run-django-hello-world/cloudbuild.yaml]: Please add steps to test the app. Basic curl ping is good enough.
You can see my dotnet PR or Adam's Go PR and adapt yours accordingly.

File: python/cloud-run-django-hello-world/.idea/modules.xml (hunk 1 above)
Comment (engelke) [reply python/cloud-run-django-hello-world/.idea/modules.xml]: I just copied this from https://github.com/GoogleCloudPlatform/cloud-code-samples/blob/master/golang/go-hello-world/.idea/modules.xml. I'm unfamiliar with IntelliJ, so I don't know whether this is right or wrong.

File: python/cloud-run-django-hello-world/cloudbuild.yaml (hunk 3 above)
Comment (engelke) [reply python/cloud-run-django-hello-world/cloudbuild.yaml]: Will do.

**Pull request merged**